packages/flutter_tools/lib/src/macos/application_package.dart

```diff
@@ -110,12 +111,13 @@ abstract class MacOSApp extends ApplicationPackage {
   String? executable(BuildMode buildMode);
 }
 
-class PrebuiltMacOSApp extends MacOSApp {
+class PrebuiltMacOSApp extends MacOSApp implements PrebuiltApplicationPackage {
   PrebuiltMacOSApp({
     required this.bundleDir,
```

Comment:
Same comment as `PrebuiltIOSApp`, `bundleDir` can be replaced by `applicationPackage`
```suggestion
```